examples/src/main/java/nl/jcore/java8demo/streaming/StreamingExecutionPlanExample.java

```diff
@@ -9,7 +9,7 @@ public static void main(String[] args) {
         final List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8);
         final List<String> output = numbers.stream()
                 .filter(i -> i % 2 == 0)
-                .sorted((i1, i2) -> i2.compareTo(i1))
+                .sorted((i1, i2) -> i2.compareTo(i1)) // Alternative: .sorted(Comparator.reverseOrder())
```

Comment:
What do you mean with 'depends on the input list'? Comparator.reverseOrder() ultimately does the same as your example, it is implemented as:

`public int compare(Comparable<Object> c1, Comparable<Object> c2) {
    return c2.compareTo(c1);
 }`

But I agree with your second comment, for lambda beginners, your implementation might be easier :)